Felienne/hedy
# [UI IMPROVEMENT] Improves UI of for-teachers page tables

**Description**
In this PR we improve the tables on the for-teachers page to be more consistent with our overall UI and more to be better understandable by users. See the screenshot below for clarification.
![Schermafbeelding 2022-07-21 om 11 35 29](https://user-images.githubusercontent.com/38155424/180182274-355040f1-a90d-4ed7-a494-dd84be49e73f.png)

**Fixes**
This PR fixes #3084.

**How to test**
Make sure you are a teacher and navigate to `/for-teachers`. Notice the new UI and verify that all elements work as expected.
